src/handbook/peopleops/job-descriptions/engineering-manager.md

```diff
@@ -0,0 +1,52 @@
+---
+navTitle: Engineering Manager
+navGroup: Job Descriptions
+---
+
+## Job Description
+
+At FlowFuse, an Engineering Manager is not just a lead to our team of engineers, but also someone who can contribute to our codebase. Further, the Engineering Manager guides their engineering team to higher performance and faster innovation.
+
+We believe in running our engineering team like a sports team. You'll be responsible for fostering a culture of high performance, continuous improvement, and a relentless drive to win.
+
+As an Engineering Manager, you'll be shaping our engineering culture and driving the execution of our product roadmap. You will:
+* **Coach and Develop Your Team**: You'll be responsible for the growth and development of your engineers. This includes conducting regular one-on-ones, providing constructive feedback, identifying areas for improvement, managing performance, and creating opportunities for career advancement. You'll know your team's strengths and weaknesses and how to best utilize their talents.
+* **Drive Performance and Velocity**: You’ll do whatever is in your power to increase team performance and velocity. You'll establish clear goals and metrics, track progress, and implement strategies to help the team move faster and more efficiently without sacrificing quality. You will analyze the team's current performance, identify bottlenecks, and implement solutions to optimize our development processes.
+* **Build a Winning Culture**: You will cultivate a high-performance, results-oriented environment. This means fostering a sense of ownership, accountability, and a shared commitment to our goals.
+* **Strategize and Execute**: You'll work closely with other stakeholders to translate business objectives into a clear technical roadmap. You will be responsible for the successful and timely delivery of user-valuable software.
```

Comment:
Bit of crossover with PM responsibilities here? I see this:

> translate business objectives into a clear technical roadmap

as a collaborative task between PM and EM. EM knows the skillset and capabilities of their team, the PM is driving the feature set and offering forward, reaching out to Stakeholders, etc.

EM role is without a doubt the latter though:

> You will be responsible for the successful and timely delivery of user-valuable software.